buildspecs/linux_riscv64.spec

```diff
@@ -0,0 +1,260 @@
+		<property name="j9dt.compileTarget" value="makefile"/>
+		<property name="j9dt.make" value="gmake"/>
+		<property name="j9dt.toolsTarget" value="buildtools.mk"/>
+		<property name="javatestPlatform" value="linux_riscv64"/>
+		<property name="jclMemoryMax" value="-Xmx64m"/>
+		<property name="jclOSStackSizeMax" value=""/>
+		<property name="jgrinderTestingSupported" value="true"/>
+		<property name="jitTestingOptLevel" value="optlevel=warm"/>
+		<property name="localRootPath" value="$(J9_UNIX_ROOT)"/>
+		<property name="longLimitCmd" value=""/>
+		<property name="main_shortname" value="rv64"/>
+		<property name="os.lineDelimiter" value="unix"/>
+		<property name="platform_arch" value="riscv64"/>
+		<property name="svn_stream" value=""/>
+		<property name="uma_make_cmd_ar" value="ar"/>
+		<property name="uma_make_cmd_as" value="as"/>
+		<property name="uma_make_cmd_cc" value="gcc"/>
+		<property name="uma_make_cmd_cpp" value="cpp -E -P"/>
+		<property name="uma_make_cmd_cxx" value="g++"/>
+		<property name="uma_make_cmd_cxx_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_dll_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_riscv_gcc_cxx" value="gcc"/>
+		<property name="uma_make_cmd_ranlib" value="ranlib"/>
+		<property name="uma_processor" value="riscv64"/>
+		<property name="uma_type" value="unix,linux"/>
+		<property name="uma_use_cflags_on_asflag_line" value="true"/>
+	</properties>
+	<features>
+		<feature id="combogc"/>
+		<feature id="core"/>
+		<feature id="crypto"/>
+		<feature id="dbgext"/>
+		<feature id="gpu"/>
+		<feature id="se"/>
+		<feature id="se60_26"/>
+		<feature id="se7"/>
+		<feature id="se70_27"/>
+	</features>
+	<source>
+		<project id="com.ibm.jvmti.tests"/>
+		<project id="compiler"/>
+	</source>
+	<flags>
+		<flag id="arch_riscv" value="true"/>
+		<flag id="build_SE6_package" value="false"/>
+		<flag id="build_autobuild" value="true"/>
+		<flag id="build_dropToHursley" value="true"/>
+		<flag id="build_dropToToronto" value="true"/>
+		<flag id="build_j2se" value="true"/>
+		<flag id="build_java8" value="false"/>
+		<flag id="build_java9" value="false"/>
+		<flag id="build_openj9" value="true"/>
+		<flag id="build_openj9JDK8" value="true"/>
+		<flag id="build_product" value="true"/>
+		<flag id="build_vmContinuous" value="true"/>
+		<flag id="env_data64" value="true"/>
+		<flag id="env_hasFPU" value="true"/>
+		<flag id="env_littleEndian" value="true"/>
+		<flag id="gc_batchClearTLH" value="true"/>
+		<flag id="gc_debugAsserts" value="true"/>
+		<flag id="gc_idleHeapManager" value="true"/>
+		<flag id="gc_inlinedAllocFields" value="true"/>
+		<flag id="gc_minimumObjectSize" value="true"/>
+		<flag id="gc_tlhPrefetchFTA" value="true"/>
+		<flag id="graph_cmdLineTester" value="true"/>
+		<flag id="graph_compile" value="true"/>
+		<flag id="graph_enableModularityTesting" value="false"/>
```

Comment:
Why is this flag disabled?

Reply:
Given that `graph_enableModularityTesting `was disabled on all other platforms, it doesn't make sense to enable it on RISC-V.

Reply:
It isn't mentioned in `linux_x86-64.spec`, but perhaps that's the same as explicitly setting it to false.
I suspect it's an obsolete flag - we can defer cleaning up such things.